File: PHP/CarBasic.php
<?php
  require_once('Car.php');
  class CarBasic extends Car {
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model) {
      parent::__construct($license, $driver);
      $this->brand = $brand;
      $this->model = $model;
    }

    public function printCarData(){
      echo "
        Licencia: $this->license 
        Conductor: {$this->driver->name} 
        Documento: {$this->driver->document}
        Marca: $this->brand
        Modelo: $this->model
        Pasajeros: $this->passengers       
      ";
    }
  }
?>

File: Java/CarBasic.java
class CarBasic extends Car{
  String brand;
  String model;

  public CarBasic(String license, Account driver, String brand, String model) {
    super(license, driver);
    this.brand = brand;
    this.model = model;
  }

  @Override
  void printCarData() {    
    super.printCarData();
    System.out.println(
      "Marca: " + brand + " - " +
      "Modelo: " + model
    );
  }
}
